grommet-leaflet/src/layers/Marker/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import * as L from 'leaflet';
+
+export interface MarkerProps {
+  children?: any;
+  icon?: React.ReactNode;
+  position?: number[];
```

Comment:
I think position is native to react-leaflet. Can we remove since we extend L.MarkerOptions?

Reply:
done